src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -62,6 +62,7 @@ public Date(TemporalAccessor date) {
     }
 
     /**
+     * CS427 Issue link: https://github.com/JabRef/jabref/issues/7864
```

Comment:
This is not ncessary

Reply:
Sorry, removed the one-line comment. 

Have a great night.